requirements-dev-py2.txt

```diff
@@ -0,0 +1,12 @@
+numpydoc >= 0.6.0
```

Comment:
yes, you need Mock in Python 2.7 and Mypy is not available for Python 2 (this library is not used at the moment, right? I guess we also should not use it to stay backward compatible).